runtime/compiler/control/CompilationThread.cpp

```diff
@@ -3778,6 +3779,26 @@ void TR::CompilationInfo::stopCompilationThreads()
          }
       }
 #endif /* defined(J9VM_OPT_JITSERVER) */
+   // check if there is any compilation logged in the global list, if so, dump the contents
+   if (J9::Options::_collectRegionLog && J9::Options::_compilationRegionLogFileName)
+      {
+      std::FILE *regionLogOutFile = fopen(J9::Options::_compilationRegionLogFileName, "w+");
```

Comment:
Why do you need `std::`?

Reply:
fopen by default will return TR::FILE but functions that takes the file descriptor to print to file in RegionLog are designed not in TR namespace. Should I put RegionLog into TR namespace? Or use TR::FILE in the function definitions there?

Reply:
I think `RegionLog` should go into the `TR::` namespace (along with `Region`) regardless of this.

Reply:
I tried to use TR:: namespce, but it turned out that fopen, fprintf, fclose only accpets std::FILE. When I use FILE* (TR::FILE *)here, the compiler complains. And I found other examples of using fopen, return value is handled in FILE (not TR::FILE) in other files in OpenJ9.
Compiler complaints:
<img width="717" alt="Screen Shot 2023-01-13 at 00 24 05" src="https://user-images.githubusercontent.com/65443542/212449916-aaffb828-645b-4a90-aa92-5beca262ee10.png">
Other cases:
<img width="795" alt="Screen Shot 2023-01-13 at 23 08 31" src="https://user-images.githubusercontent.com/65443542/212449985-5f2623b1-139a-4ef8-9f07-a3942a9bd453.png">


Another thing I discovered is that I'll need J9::IO::fopen to get TR::FILE, is this the correct way to do? And if I do it this way, I feel I have to find a counterpart in OMR that does the same and takes TR::FILE as well to substitute all of my fprintfs.

Reply:
Currently I keep this only one std::FILE and compilation is ok. And I haven't seen a use case of J9::IO::fopen() besides its definition or any file that includes the header file for J9IO.hpp.

Reply:
Oh, probably all of them includes IO.hpp, I'll try using that instead.

Reply:
Changed to using J9::IO::func in OpenJ9 and TR::IO::func in OMR. Compilation OK